app/sqs_config.go

```diff
@@ -43,8 +43,10 @@ var DefaultConfig = domain.Config{
 		CacheExpiryMs:          2000, // 2 seconds.
 		DefaultQuoteHumanDenom: "usdc",
 
-		MaxPoolsPerRoute: 4,
-		MaxRoutes:        5,
-		MinOSMOLiquidity: 50,
+		MaxPoolsPerRoute:       4,
+		MaxRoutes:              5,
+		MinOSMOLiquidity:       50,
+		CoingeckoUrl:           "https://prices.osmosis.zone/api/v3/simple/price",
+		CoingeckoQuoteCurrency: "usd",
```

Comment:
Could we also update `config_testnet.json` with the new values?

Reply:
Done